python/cudf/cudf/pylibcudf_tests/test_avro.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import io
+import itertools
+
+import fastavro
+import pyarrow as pa
+import pytest
+from utils import assert_table_and_meta_eq
+
+import cudf._lib.pylibcudf as plc
+
+avro_dtype_pairs = [
+    ("boolean", pa.bool_()),
+    ("int", pa.int32()),
+    ("long", pa.int64()),
+    ("float", pa.float32()),
+    ("double", pa.float64()),
+    ("bytes", pa.string()),
+    ("string", pa.string()),
+]
+
+
+@pytest.fixture(
+    scope="module",
+    params=itertools.product(avro_dtype_pairs, avro_dtype_pairs),
+)
+def avro_dtypes(request):
+    return request.param
+
+
+@pytest.fixture
+def avro_dtype_data(avro_dtypes):
+    # avro_type1, _, avro_type2, _ = avro_dtypes
+    types1, types2 = avro_dtypes
+    avro_type1, _ = types1
+    avro_type2, _ = types2
```

Comment:
If you want to destructure in one go:

```
(type1, _), (type2, _) = avro_dtypes
```

Reply:
Thanks, didn't know you could do that!

🤯